.gitignore

```diff
@@ -1,4 +1,5 @@
-
+*.pyc
+*.pyo
```

Comment:
Not during compilation but during test execution. I agree that it shouldn't impact the sources. However, I see only 3 ways of doing that:

1- copy the whole test suite to the build tree
2- adding "-B" to the python command in order for python not to generate any bytecode
3- ignoring .pyc and .pyo

That 3rd solution is the most convenient.